content/guides/tasks/for-file.md

```diff
@@ -0,0 +1,18 @@
+---
+rank: 6
+related_endpoints:
+  - get_files_id_tasks
+related_guides:
+required_guides: []
+related_resources:
+  - files
+  - tasks
+alias_paths: []
+---
+
+# Lists all tasks for a file
+
+To list all of the tasks for a specific file, call the `GET
+/files/:file_id/tasks` with the file `id`.
```

Comment:
We should link to the API reference here.

Reply:
The file or task API reference?

Reply:
The reference for that endpoint. https://developer.box.com/en/reference/get-files-id-tasks/